numpy/core/meson.build

```diff
@@ -53,7 +53,11 @@ C_API_VERSION = '0x00000012'
 
 # Check whether we have a mismatch between the set C API VERSION and the
 # actual C API VERSION. Will raise a MismatchCAPIError if so.
-run_command('code_generators/verify_c_api_version.py', '--api-version', C_API_VERSION, check: true)
```

Comment:
```
~/os/numpy (enh_24034*) » tail /home/ganesh/os/numpy/build/meson-logs/meson-log.txt                                                                                         ganesh@ganesh-MS-7B86

--- stderr ---
Traceback (most recent call last):
  File "/home/ganesh/os/numpy/numpy/core/code_generators/verify_c_api_version.py", line 66, in <module>
    raise ValueError("testing")
ValueError: testing



numpy/core/meson.build:56:0: ERROR: Command `/home/ganesh/os/np-test/bin/python3 /home/ganesh/os/numpy/numpy/core/code_generators/verify_c_api_version.py --api-version 0x00000012` failed with status 1.
```

The logs have it. But if you need it in stdout during `spin`, the CR diff seems to be the best way.

Reply:
Ah good to know. I personally think it’s more convenient to see the new hash in the build output when this script fails so I can just copy/paste the new hash into `cversions.txt` rather than remembering that it’s in the logs and finding them.